test/FunctionsV2/StorageProviderSelectionTests.cs

```diff
@@ -4,77 +4,174 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using DurableTask.Core;
 using Microsoft.Azure.WebJobs.Extensions.DurableTask;
 using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
-using Microsoft.Azure.WebJobs.Host.TestCommon;
 using Moq;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace WebJobs.Extensions.DurableTask.Tests.V2
 {
     public class StorageProviderSelectionTests
     {
-        static StorageProviderSelectionTests()
-        {
-            AzureStorageDurabilityProviderFactoryMock = new Mock<IDurabilityProviderFactory>();
-            AzureStorageDurabilityProviderFactoryMock.Setup(f => f.Name).Returns("AzureStorage");
+        public StorageProviderSelectionTests() { }
 
-            MicrosoftSQLDurabilityProviderFactoryMock = new Mock<IDurabilityProviderFactory>();
-            MicrosoftSQLDurabilityProviderFactoryMock.Setup(f => f.Name).Returns("MicrosoftSQL");
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        [InlineData("AzureStorage")]
```

Comment:
If you are only going to use one piece of data, don't parameterize the method and just use a constant.

Reply:
That being said, most of the code is identical for these tests, so I think you can parameterize them.

All you need to do is modify your verify statements to look something like this.

```csharp
azurestorageMock.Verify(a => a.GetDurabilityProvider(), string.Equals(storageProvider, "AzureStorage") ? Times.Once() : Times.Never())
```